src/app/lib/utilities/preprocessor/rules/cpsAsset/convertCpsBlocksToOptimoBlocks/index.js

```diff
@@ -0,0 +1,75 @@
+import { clone, pathOr } from 'ramda';
+import image from './generators/image';
+import {
+  candyXmlToRichText,
+  stringToRichText,
+} from '../../../../../../../../../fabl-modules/libraries/rich-text-transforms';
+
+const wrapText = innerXML => `<body><paragraph>${innerXML}</paragraph></body>`;
+
+const handleCandyXML = xml => candyXmlToRichText(xml);
```

Comment:
We need this anonymous wrapper function?  Isn't `handleCandyXML` the same as `candyXmlToRichText`